packages/expo-cli/src/commands/client/index.js

```diff
@@ -12,8 +12,14 @@ import log from '../../log';
 import prompt from '../../prompt';
 import urlOpts from '../../urlOpts';
 import { PLATFORMS } from '../build/constants';
+<<<<<<< HEAD
```

Comment:
merge conflict

Reply:
my bad, it was already fixed in [dc250a5](https://github.com/expo/expo-cli/pull/1148/commits/dc250a5c70e985d8385bf5f202e45412f1d26ee2) but my server didn't push it to the right remote.